core/src/avm1/globals/text_field.rs

```diff
@@ -859,3 +860,36 @@ fn set_filters<'gc>(
     this.set_filters(activation.context.gc_context, filters);
     Ok(())
 }
+
+fn restrict<'gc>(
+    this: EditText<'gc>,
+    activation: &mut Activation<'_, 'gc>,
+) -> Result<Value<'gc>, Error<'gc>> {
+    match this.restrict() {
+        Some(value) => Ok(AvmString::new(activation.context.gc_context, value).into()),
+        None => Ok(Value::Null),
+    }
+}
+
+fn set_restrict<'gc>(
+    this: EditText<'gc>,
+    activation: &mut Activation<'_, 'gc>,
+    value: Value<'gc>,
+) -> Result<(), Error<'gc>> {
+    match value {
+        Value::Undefined | Value::Null => {
+            this.set_restrict(None, &mut activation.context);
+        }
+        _ => {
+            let text = value.coerce_to_string(activation)?;
+            if text.is_empty() {
+                // According to docs, an empty string means that you cannot enter any character,
+                // but according to reality, an empty string is equivalent to null in AVM1.
```

Comment:
This is something that could be added to the spec errata in the Wiki perhaps.
We find lots of stuff like this all the time, and don't put half of it in there, but still...

Reply:
Added this information, hope it's okay: https://github.com/ruffle-rs/ruffle/wiki/SWF-AVM-Specification-Errata

Reply:
It's perfect, thank you!